doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -229,6 +229,27 @@ Securing Solr
 
 Our sample init script and systemd service file linked above tell Solr to only listen on localhost (127.0.0.1). We strongly recommend that you also use a firewall to block access to the Solr port (8983) from outside networks, for added redundancy.
 
+
+To start Solr Apache in external use mode, you must access the Solr example directory:
+
+# cd /usr/local/solr-7.7.2/bin/
+
+# ./solr start -j "-Djetty.host = DNS” -force
+
+Right after that you should use the iptables blocking rule
+
+# iptables -A INPUT -s ip address / 32 -p tcp -m tcp --dport 8983 -j ACCEPT
+
+# iptables -A INPUT -p tcp -m tcp --dport 8983 -j DROP
+
+As a result, your application will reject Solr's external access, assuring connections only through Dataverse.
+
+If you want to upload Solr Apache without a GUI prompt, use it as follows:
+
+# ./solr start -j "-Djetty.host = hostaname" -noprompt
+
+With that application will work on localhost without access to GUI.
```

Comment:
I'm not sure this part about the GUI is necessary. If you're saying it's related to securing Solr, you might want to mention this. Finally, do you mean "upload"? Maybe "access" instead? Or "use"?

Reply:
When the "./solr start -j" part is mentioned -Djetty.host = hostaname "-noprompt"

it is more for security reasons in the local environment so that the user does not access the solr locally and make undue changes.